apps/settings/js/panels/about_more_info/hardware_info.js

```diff
@@ -52,6 +52,8 @@ define(function(require) {
      */
     _refreshBluetoothAddress: function mi__refreshBluetoothAddress(address) {
       // update UI fields
+      if (address===null || address==="")
+        address="Bluetooth is disabled.";
```

Comment:
Because we need support multiple language, please add a string identifier in `locales/settings.en-US.properties` instead of embed string in code.